### ISSUE TITLE
Hey, Want To Know A Little Bit.

### ISSUE THREAD

**Issue by Th3-C0der** [milutinke/Ultimate-Anti-Cheat-UAC#4]
Hi, I Just Want To Know How I Can Get Cs 1.6 Cod MW4 Mod ByM.
Sorry, I Don't Know Where To Contact You So I Am Creating A New Issue Here On GitHub.
Thanks In Advance.

**Comment by milutinke** [comment]
I can sell it to you for 100 EUR without the source code and without any support.
I do not develop or maintain AMXX plugins for 4 years already.

**Comment by Th3-C0der** [comment]
Are u kidding, u mean 100 EUR without the source code?
I am just a normal guy/teen & fan of Cod MW4 mod(Spent Playing More Than 1000 hours playing it on a server back in lockdown(2020-2021)). and I don't have money to buy the mod 🥲.
Anyways Thanks For Answering 🫡.

**Comment by milutinke** [comment]
> Are u kidding, u mean 100 EUR without the source code? I am just a normal guy/teen & fan of Cod MW4 mod(Spent Playing More Than 1000 hours playing it on a server back in lockdown(2020-2021)). and I don't have money to buy the mod 🥲. Anyways Thanks For Answering 🫡.

I spent over a hundred hours making it perfect, all from scratch, not giving it away for free lol

**Comment by Th3-C0der** [comment]
Yes, I Know You Have Spent A Lot Of Time In Making It Perfect & Better,
You Made As Amazing Mod, And I Have A Good Memories Playing The Mod.
I Would Like To Thank You For Creating Such A Masterpiece.
And Also Thanks For Answering The Question.
You're A Great Person 🫡.
